### PR TITLE
REF: Plausible deniability uses Hooks

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jga9bfeh094fravpfvnbz139jr

[download apk](http://filestorage.bluewallet.io:1488/1606035145.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/p2dkpptfhc69kw2nbrd2hnjy04

[download apk](http://filestorage.bluewallet.io:1488/1606045251.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jc9jjw0kttwvm2ag8rjkhcj3r4

[download apk](http://filestorage.bluewallet.io:1488/1606068984.apk)

File: loc/en.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
     "help": "Under certain circumstances, you might be forced to disclose a password. To keep your coins safe, BlueWallet can create another encrypted storage, with a different password. Under pressure, you can disclose this password to a 3rd party. If entered in BlueWallet, it will unlock a new 'fake' storage. This will seem legit to a 3rd party, but it will secretly keep your main storage with coins safe.",
     "help2": "The new storage will be fully functional, and you can store some minimum amounts there so it looks more believable.",
     "password_should_not_match": "Password is currently in use. Please, try a different password.",
-    "passwords_do_not_match": "Passwords do not match, try again",
+    "passwords_do_not_match": "Passwords do not match, please try again.",
```
Comment (Overtorment) [inline loc/en.json:124]: why "please"? if you dont want it  - piss off  mate 😂

File: screen/plausibledeniability.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 /* global alert */
-import React, { Component } from 'react';
+import React, { useContext, useState } from 'react';
```
Comment (Overtorment) [inline screen/plausibledeniability.js:2]: its a good thing plausible deniability is covered with e2e

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rp4keukhedtubahfuwp5cjcv94

[download apk](http://filestorage.bluewallet.io:1488/1606185839.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ep30q0g0e0f0xu64yw9pbb6ktr

[download apk](http://filestorage.bluewallet.io:1488/1606188463.apk)

**Review by ncoelho — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**